src/intersections/raytriangle.jl

```diff
@@ -49,7 +56,15 @@ function intersection(f, r::Ray{3,T}, t::Triangle{3,T}) where {T}
     return @IT NoIntersection nothing f
   end
 
+  any(o .≈ vs) && (return @IT CornerOriginTouchingRayTriangle r(λ) f)
```

Comment:
all these cases where we use isapprox we also need to use atol(T) for the correct precision.

Reply:
I'm not sure that is necessary. `isapprox` finds the least precise type and sets the tol based on that.

Reply:
It is because we use `isapprox(a, b, atol=atol(T))` with the function `atol(T)` defined in the package. We have tolerances for 64 and 32 bits that are not necessarily the same as default tolerances in numerical code.

Reply:
Oh, ok. Can you link where `atol` is defined in the package just so I know what we are doing here?

Reply:
Oh, I see. What is the purpose of this? There is a good reason I;m sure, but this should be stated in the documentation (unless it is and I missed it) because its not obvious to a contributor/user.

Reply:
I think we just fixed a precision in the source and used it consistently. If at some point we need to change this precision we can by changing it in a single place.